### ISSUE TITLE
Fix false positives for overflowX for CSS-in-JS in property-no-unknown

### ISSUE THREAD

**Issue by ivancuric** [stylelint/stylelint#4020]
> Clearly describe the bug

Using React and the CSS object notation leads to an error for the following piece of code:
```jsx
<div style={{overflowX: 'scroll', color: 'black'}}>
```
error:
```shell
demo.jsx
68:41  ×  Unexpected unknown property "overflowx"   property-no-unknown
```
> Which rule, if any, is the bug related to?

`property-no-unknown`

> What CSS is needed to reproduce the bug?

```js
style={overflowX: 'scroll'}
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "extends": ["stylelint-config-standard", "stylelint-config-prettier"],
}
```

> Which version of stylelint are you using?

`9.10.1`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI, VScode

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```shell
demo.jsx
68:41  ×  Unexpected unknown property "overflowx"   property-no-unknown
```

**Comment by jeddy3** [comment]
@ivancuric Thanks for the report and for using the template.

I can reproduce this using the demo using:

```
export default (<div style={{ overflowX: "hidden" }}></div>)
```

`overflowY` also produces a violation, whereas `overflowWrap` doesn't. I suspect the issue is restricted to single character hyphenated property names and is therefore likely connect with [this previous bug](https://github.com/gucong3000/postcss-jsx/issues/32) is postcss-jsx.

As the issue has been labelled "help wanted", please consider [contributing a fix](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#fixing-a-bug-in-an-existing-rule) upstream.

**Comment by vankop** [comment]
I take this

**Comment by vankop** [comment]
@gucong3000 @hudochenkov review please ⬆

**Comment by evilebottnawi** [comment]
@vankop can we add test for this case to avoid regressions in future?

**Comment by brandonkal** [comment]
Ran into this as well. Please push a stylelint update with these fixes.

**Comment by vankop** [comment]
@brandonkal I think you just need to reinstall stylelint since it was fixed in `postcss-jsx` patch version and stylelint does not use `lock` file

**Comment by hudochenkov** [comment]
True. Reinstalling stylelint should fix the problem. `npm ls postcss-jsx` should show 0.36.3 (version with the fix).

**Comment by brandonkal** [comment]
Thanks guys! I had the latest version of `postcss-jsx`. Turns out I had to restart my editor to get the updated stylelint  to register in the problems panel.
